src/components/form/select/styles.css

```diff
@@ -0,0 +1,14 @@
+.Select select {
```

Comment:
Maybe it's a good idea to start using sass, it helps you write cleaner css code.